src/Emulator/Instructions/Instructions.css

```diff
@@ -0,0 +1,19 @@
+.instructions {
```

Comment:
Assuming that padding and such happens later in individual components

Reply:
Yup! Currently, each component has its own folder, which contains the JS file for the component as well as the stylesheet for that component.

Reply:
Neat. I like the weather app model - thanks for clarifying.